### ISSUE TITLE
Mask value not registered on first typed value

### ISSUE THREAD

**Issue by vlaca99** [uNmAnNeR/imaskjs#342]
When I type a value in input, mask value is " " on first typed character and gets value only after second typed character.
e.g. I type 1234, logged mask value is 123

![mask-value](https://user-images.githubusercontent.com/25773880/78890881-bdee8600-7a66-11ea-905b-31ab1b4ef8ac.png)

So when i add a value in one input and move directly to another it ads value from previous input.
![image](https://user-images.githubusercontent.com/25773880/78891021-f68e5f80-7a66-11ea-9b43-b90240eef740.png)

**Comment by uNmAnNeR** [comment]
hi! @vlaca99 can you provide some fiddle to run?

**Comment by vlaca99** [comment]
Hi @uNmAnNeR ! Thank you for your response. 
Fortunately I've made it work for me regarding this issue.
However there is another issue that I came across and will post it here.
